.github/workflows/analysis_workflow.yml

```diff
@@ -96,6 +96,8 @@ jobs:
           share/ci/scripts/linux/install_openexr.sh latest
           share/ci/scripts/linux/install_imath.sh latest
```

Comment:
This will pre-install the latest Imath tag, which we are then doing again below (to a different location) with `-DOCIO_INSTALL_EXT_PACKAGES=ALL` and `-DOCIO_INSTALL_EXT_PACKAGES_LATEST=ON`. We will also be installing expat, lcms2, yaml-cpp, pystring, pybind11, and openfx twice with this current setup. Previously we used `-DOCIO_INSTALL_EXT_PACKAGES=NONE` for this workflow because these were all pre-installed with the scripts. If `-DOCIO_INSTALL_EXT_PACKAGES_LATEST=ON` is preferred, I would suggest only using the above install scripts for optional dependencies (and perhaps their dependencies, which may overlap with OCIO) which aren't installed by OCIO. Or, use `-DOCIO_INSTALL_EXT_PACKAGES=MISSING` to avoid duplicates where possible.

Reply:
Perhaps the two defines approach is not the optimal one. To implement the right behavior, the code could support a fourth option dedicated to `CI latest` ?

Reply:
I could remove some of the dependencies install script but not much because OIIO need quite a lot of what OCIO already uses. I can remove all remaining `OCIO_INSTALL_EXT_PACKAGES_LATEST` stuff and only rely on install script, hoping that they will work on all 3 platforms (to that end we may want to move them to some non platform dependent sub folder as it seems it would only be copy and paste port). The advantage of `OCIO_INSTALL_EXT_PACKAGES_LATEST` was that it doesn't require any additional scripts to work but we already removed most of that and there might not be any use case out of this analysis workflow file.

Reply:
I'll try to have something ready this week to address this point.

Reply:
I pushed an update implementing the requested, change. As before analysis workflow temporarily enabled on PR to validate the changes.